Code/client/Games/Skyrim/Forms/MagicItem.cpp

```diff
@@ -31,6 +31,17 @@ bool MagicItem::IsBuffSpell() const noexcept
     }
 }
 
+bool MagicItem::IsBoundWeaponSpell() noexcept
+{
+    for (EffectItem* pEffect : listOfEffects)
+    {
+        if (pEffect->pEffectSetting->eArchetype == EffectArchetypes::ArchetypeID::kBoundWeapon)
```

Comment:
are we sure pEffectSetting is always valid?

Reply:
Add nullptr checks